streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsMetricsImplTest.java

```diff
@@ -68,46 +64,46 @@ public void testRemoveSensor() {
         streamsMetrics.removeSensor(sensor3);
     }
 
-    @Test
-    public void testLatencyMetrics() {
-        String groupName = "doesNotMatter";
-        String scope = "scope";
-        String entity = "entity";
-        String operation = "put";
-        Map<String, String> tags = new HashMap<>();
-        StreamsMetricsImpl streamsMetrics = new StreamsMetricsImpl(new Metrics(), groupName, tags);
-
-        Sensor sensor1 = streamsMetrics.addLatencyAndThroughputSensor(scope, entity, operation, Sensor.RecordingLevel.DEBUG);
-
-        Map<MetricName, ? extends Metric> metrics = streamsMetrics.metrics();
-        // 2 meters and 4 non-meter metrics plus a common metric that keeps track of total registered metrics in Metrics() constructor
-        int meterMetricsCount = 2; // Each Meter is a combination of a Rate and a Total
-        int otherMetricsCount = 4;
-        assertEquals(meterMetricsCount * 2 + otherMetricsCount + 1, metrics.size());
-
-        streamsMetrics.removeSensor(sensor1);
-        metrics = streamsMetrics.metrics();
-        assertEquals(metrics.size(), 1);
-    }
-
-    @Test
-    public void testThroughputMetrics() {
-        String groupName = "doesNotMatter";
-        String scope = "scope";
-        String entity = "entity";
-        String operation = "put";
-        Map<String, String> tags = new HashMap<>();
-        StreamsMetricsImpl streamsMetrics = new StreamsMetricsImpl(new Metrics(), groupName, tags);
-
-        Sensor sensor1 = streamsMetrics.addThroughputSensor(scope, entity, operation, Sensor.RecordingLevel.DEBUG);
-
-        Map<MetricName, ? extends Metric> metrics = streamsMetrics.metrics();
-        int meterMetricsCount = 2; // Each Meter is a combination of a Rate and a Total
-        // 2 meter metrics plus a common metric that keeps track of total registered metrics in Metrics() constructor
-        assertEquals(meterMetricsCount * 2 + 1, metrics.size());
-
-        streamsMetrics.removeSensor(sensor1);
-        metrics = streamsMetrics.metrics();
-        assertEquals(metrics.size(), 1);
-    }
+//    @Test
```

Comment:
For `testLatencyMetrics` and `testThroughputMetrics` maybe use `@Ignore` instead ? Not a big deal but by getting an `ignored` test count there's a better chance these two tests won't fall through the cracks.

Reply:
ah, that's how you do it. I tried `@Test(ignore=true)` like testng, but that obviously doesn't work...